static/js/formHandler.js

```diff
@@ -115,6 +122,66 @@ const formSubmitHandler = () => {
         });
 };
 
+function voltageChartAjax(inputs, toGraph){
```

Comment:
Can we add this as an onClick for each radio button so the user doesn't have to press calculate to see the new graph?